README.md

```diff
@@ -1,70 +1,63 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Math-Magician
+> This is the project of Calculator Application where I have used React and it's necessary tools.
 
-## Available Scripts
+![screenshot](./src/images/screenshot.png)
 
-In the project directory, you can run:
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Built With
 
-### `npm test`
+- HTML, CSS, js, React
+- Linters, Markdown
+- GitHub Flow, GitHub Action
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+ ## Getting Started
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+To get a local copy up and running follow these simple example steps.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+1) Enter `git@github.com:basnetlaxmi/math-magician.git` into terminal
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+2) cd math-magician directory
+3) npm start
+4) View in the browser
```

Comment:
- [ ] Kindly include instructions on how to install your project dependencies. So after number `2` which is `cd math-magician directory,` you can add `npm install` before `npm start`